repository-data/application/src/main/resources/hcm-config/configuration/translations/types.yaml

```diff
@@ -15,11 +15,39 @@ definitions:
       /en:
         jcr:primaryType: hipposys:resourcebundle
         jcr:name: '[Page] Landing page'
+        hee:tags: Keywords
+        hee:tagssupport: Keyword suggestions
```

Comment:
Could we remove `hee:tagssupport` translation for all page document types (as apparently, it doesn't override the UI label/caption for the field)?

I think we may need to override it via page document type editor template configuration (e.g. `/hippo:namespaces/hee/<page_document_type>/editor:templates/_default_/tagsuggest/@caption='Tag suggestions'`) or override `hippo-plugin-content-tagging-14.7.13.jar!/org/onehippo/forge/ecmtagging/editor/TagSuggestPlugin.properties` resource bundle.

But, we may not be able to override the `Refresh keywords` label via the document type editor template configuration and so we may need to do it by overriding `hippo-plugin-content-tagging-14.7.13.jar!/org/onehippo/forge/ecmtagging/editor/TagSuggestPlugin.properties` resource bundle.